app/routes/publications/overview/shortlist.js

```diff
@@ -0,0 +1,33 @@
+import Route from '@ember/routing/route';
+import fetch from 'fetch';
+import { inject as service } from '@ember/service';
+
+export default class PublicationsOverviewShortlistRoute extends Route {
+  @service store;
+
+  async model() {
+    const endpoint = '/publication-flows/shortlist';
+    const response = await fetch(endpoint, {
+      Headers: {
+        'Accept': 'application/vnd.api+json',
+      }
+    });
+    const result = await response.json();
+
+    if (result.data.length) {
+      return this.store.query('piece', {
+        include: [
+          'agendaitems.agenda.next-version',
+          'agendaitems.mandatees.person',
+          'agendaitems.treatment.decision-activity',
+          'document-container.type',
+        ].join(','),
+        sort: '-created',
+        'page[size]': result.data.length,
+        'filter[:id:]': result.data.map((record) => record.id).join(','),
+      });
+    }
```

Comment:
Potentially loads too much data (all mandatees for all agendaitem versions linked to 1 piece)
But we only need the latest agendaitem.
Perhaps this one should be simpler with just including `document-container.type`.
We could then just rely on the getter (waiting in the template) or load some data in the afterModel instead

Reply:
Same remark as above. We can remove the includes (but with new mu-cl-resources this call is relatively fast), but that also means we're going to be making a lot of calls when rendering. Best to check if this call is fast enough first (it is, on my machine).